src/plugins/files_management/public/i18n_texts.ts

```diff
@@ -0,0 +1,71 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+import { i18n } from '@kbn/i18n';
+
+export const i18nTexts = {
+  titleColumnName: i18n.translate('filesManagement.table.titleColumnName', {
+    defaultMessage: 'Name',
+  }),
+  tableListTitle: i18n.translate('filesManagement.table.title', { defaultMessage: 'Files' }),
+  entityName: i18n.translate('filesManagement.entityName.title', { defaultMessage: 'file' }),
+  retry: i18n.translate('filesManagement.button.retry', {
+    defaultMessage: 'Retry',
+  }),
+  entityNamePlural: i18n.translate('filesManagement.entityNamePlural.title', {
+    defaultMessage: 'files',
+  }),
+  emptyPromptTitle: i18n.translate('filesManagement.emptyPrompt.title', {
+    defaultMessage: 'No files found',
+  }),
+  emptyPromptDescription: i18n.translate('filesManagement.emptyPrompt.description', {
+    defaultMessage: 'Any files created in Kibana will be listed here.',
+  }),
+  size: i18n.translate('filesManagement.table.sizeColumnName', {
+    defaultMessage: 'Size',
+  }),
+  diagnosticsFlyoutTitle: i18n.translate('filesManagement.diagnostics.flyoutTitle', {
+    defaultMessage: 'Diagnostics',
+  }),
+  failedToFetchDiagnostics: i18n.translate('filesManagement.diagnostics.errorMessage', {
+    defaultMessage: 'Could not fetch diagnostics',
```

Comment:
What can the user do here to fix the problem? Can you add some text?

Reply:
Just below this copy, we'll show a button that says "Retry", like the following:

<img width="434" alt="image" src="https://user-images.githubusercontent.com/8155004/200567790-2745f565-368c-42f4-867b-cd7df2f892e5.png">